pkg/tests/api/alerting/api_notification_channel_test.go

```diff
@@ -2118,6 +2118,7 @@ var expNonEmailNotifications = map[string][]string{
 	"slack_recv1/slack_test_without_token": {
 		`{
 		  "channel": "#test-channel",
+			"text": "Integration Test [FIRING:1] SlackAlert1 ",
```

Comment:
Please fix the indentation.

Reply:
Thank you for your feedback. This has been adjusted.